### ISSUE TITLE
The latest update for OBS Back ground removal will not load because of Microsoft Defender SmartScreen

### ISSUE THREAD

**Issue by FFNOP** [occ-ai/obs-backgroundremoval#559]
The latest update for OBS Back ground removal will not load because of this: Microsoft Defender SmartScreen prevented an unrecognized app from starting. Running this app might put your PC at risk.

I have tried top install the update but this is all I get and so I was wondering if there is anything that can be done to fix this so that people who might be dealing with this same issue can get help.  The full message I keep getting is the following:

Windows protected your PC
Microsoft Defender SmartScreen prevented an unrecognized app from starting. Running this app might put your PC at risk.

App: 
obs-backgroundremoval-1.1.12-windows-x64-Installer.exe 
Publisher:  
Unknown publisher

So if there is ANYTHING that can be done to fix the update so that the above stops happening (use to be it gave the option to say yes install anyway but this time no such luck.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Log and Crash Report**

Upload your OBS log file here (`Help`>`Log Files`>`Show Log Files`) to help us find cause of the problem.

If OBS is crashing, upload the crash report here as well (`Help`>`Crash Reports`>`Show Crash Reports`).

**Desktop (please complete the following information):**

 - OS: \[e.g. iOS\]
 - Browser: \[e.g. chrome, safari\]
 - Plugin Version: \[e.g. 0.5.1\]
 - OBS Version: \[e.g. 28.02\]

**Additional context**

**Comment by royshil** [comment]
@FFNOP sorry about that - it happens with every new release... microsoft just doesn't "trust" it or something. I am submitting it for inspection with them now.
you can disable Defender temporarily just for the download and install. that's simple to do in "Windows Security" on your PC - just turn the smartscreen off for a moment to install.

**Comment by royshil** [comment]
![image](https://github.com/occ-ai/obs-backgroundremoval/assets/441170/2a874843-c723-428b-aaf7-60e76ddba1e2)
IDK.. Microsoft has cleared the file. So it should be just a couple hours and it will clear.

If i can get $200 I can sign the .exe! and then all these problems would go away.

**Comment by FFNOP** [comment]
Unfortunately I attempted to do that but it's done through my antivirus
software so I got to try that again, but I'll give it a shot see if I can
disable it. I have the downloading question. I just have to be able to
install beyond that. Usually it works pretty good.

**Comment by royshil** [comment]
@FFNOP Yes every new version release needs to build "trust" so for the first few people downloading it's going to trigger a warning. It usually goes away after a day or two

**Comment by FFNOP** [comment]
Okay so I've already got it downloaded so if I didn't say another day or
two it should it should work. Is that my understanding? I mean I hope
because you know that background part of OBS really helps with my podcast a
lot. So if that's the case, I'll wait till at least Sunday and try again.
If that doesn't work got any suggestions cuz I've tried everything I know
how to do and it's not working so there you go.
